packages/react-avatar/etc/react-avatar.api.md

```diff
@@ -4,15 +4,29 @@
 
 ```ts
 
-import type { ComponentPropsCompat } from '@fluentui/react-utilities';
+import { ComponentPropsCompat } from '@fluentui/react-utilities';
```

Comment:
It looks like ComponentPropsCompat is still an interface, while ComponentStateCompat and the other exports from [types.ts](https://github.com/microsoft/fluentui/blob/55ed895d79e490765f9bc6d02f660fa312ee24e8/packages/react-utilities/src/compose/types.ts#L118) are types, is this intentional?

Reply:
Tho also maybe this is moot since we should move off *Compat anyway?

Reply:
I was waiting on Ben to confirm that the react-utilities/compose are only used by vNext.  I will update these as well.